content/en/releases/_index.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Releases"
+linkTitle: "Core Framework"
```

Comment:
I think this should be "Releases" as the left hand nav is a bit confusing with so many "CHT"s and "Core"s:

```suggestion
linkTitle: "Releases"
```

![image](https://github.com/user-attachments/assets/c17613f3-92f9-485d-8dfb-8ae7fbd6956e)